main.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  script       = "launch_ranchhand.sh"
+  ip_addresses = join(",", var.node_ips)
+  ansbile_ssh_proxy = var.ssh_proxy_host == "" ? "" : format("-o StrictHostKeyChecking=no -o ProxyCommand=\"ssh -o StrictHostKeyChecking=no -W %%h:%%p -q %s@%s\"", var.ssh_proxy_user, var.ssh_proxy_host)
+  cert_dnsnames = format("DNS:%s", join(",DNS:", var.cert_dnsnames))
+  cert_ipaddresses = length(var.cert_ipaddresses) == 0 ? "" : format(",IP:%s", join(",IP:", var.cert_ipaddresses))
```

Comment:
[formatlist](https://www.terraform.io/docs/configuration/functions/formatlist.html) + join for `cert_dnsnames` and `cert_ipaddresses`?